docs/source/quickstart.mdx

```diff
@@ -0,0 +1,545 @@
+      className="screenshot"
+      width="500"
+      alt="A dialog from GraphOS Studio displays a hidden API key and graph ref."
+    />
+
+    - The protected value that appears after `APOLLO_KEY=` and beginning with `service:` is your graph API key.
+    - The value that appears after `rover subgraph publish` is your graph ref. By default, it ends with `@current`.
+
+    <Caution>
+
+      API keys are secret credentials. Never share them outside your organization or commit them to version control. Delete and replace API keys that you believe are compromised.
+
+    </Caution>
+
+4. Leave the dialog open; you'll complete setup with Rover.
+
+#### Publish subgraph schemas
+
+1. Ensure your subgraphs are still [running locally](#run-subgraphs).
+2. Take the example code block you copied from Studio and replace the last three lines with the following:
+
+    ```bash showLineNumbers=false
+        --schema ./subgraphs/products/schema.graphql \
+        --name products \
+        --routing-url http://localhost:4001/products/graphql
+    ```
+
+    It should look something like the following, with the `<GRAPH_API_KEY>` and `<GRAPH_REF>` replaced with your graph's values:
+
+    ```bash showLineNumbers=false
+    APOLLO_KEY=service:<GRAPH_API_KEY> \
+    rover subgraph publish <GRAPH_REF> \
+    --schema ./subgraphs/products/schema.graphql \
+    --name products \
+    --routing-url http://localhost:4001/products/graphql
+    ```
+
+    - The `--schema` option specifies the path to the schema file to publish. 
+    - The `--name` option specifies the name of the subgraph.
+    - The `--routing-url` is the address the router should send requests to.
+    
+    <Note>
+
+    In a production supergraph, a subgraph's routing URL is generally its public endpoint.
+
+    </Note>
+
+2. In a new terminal window, paste and run your multi-line command.
+    - Because the command provides a `localhost` address for the `routing-url`, the terminal will confirm that you want to publish. Enter `Y`.
+    - If the command is successful, you'll see the following in your terminal:
+
+      ```bash showLineNumbers=false disableCopy=true
+      A new subgraph called 'products' was created in <GRAPH_REF>
+      The supergraph schema for <GRAPH_REF> was updated,
+      composed from the updated 'products' subgraph
+      ```
+
+    - After a few moments, the GraphOS Studio dialog you left open when creating an empty supergraph will show **Schema published** on the bottom left.
+
+    <img
+      src="./images/quickstart/schema-published.jpg"
+      className="screenshot"
+      width="500"
+      alt="A dialog from GraphOS Studio updates to show a schema has been published."
+    />
+
+3. Run the `rover subgraph publish` command for the `inventory` subgraph, substituting your graph ref. You no longer need to include the first line containing your graph API key:
+
+    ```bash showLineNumbers=false
+    rover subgraph publish <GRAPH_REF> \
```

Comment:
having the graphref auto-populated in those snippets would be awesome. can't wait for the new platform.
we could add an input for them to specify the graph ref once they created it so that we can update all the snippets?

Reply:
> we could add an input for them to specify the graph ref once they created it so that we can update all the snippets?

Hmm, like an input in the docs page itself? Not sure it's worth the extra effort in this case since that means readers have an extra copy and paste, and then only "benefit" by copying directly from the docs twice more.

But agree, autopopulating the graph ref on the next platform will be really nice!